spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -185,9 +177,66 @@ public Object run() {
 		return null;
 	}
 
+	protected PoolingHttpClientConnectionManager newConnectionManager() {
```

Comment:
Why did you move this as opposed to just removing static? I can't tell if you've made changes or just moved it.